.github/workflows/main.yml

```diff
@@ -692,7 +692,7 @@ jobs:
     # `unsafe_disable_continuation_linearity_check` makes the test
     # `cont_twice` fail.
     - run: |
-        (cargo test --features=unsafe_disable_continuation_linearity_check --test wast -- --exact Cranelift/tests/misc_testsuite/typed-continuations/cont_twice.wast && test $? -eq 101) || test $? -eq 101
+        (cargo run --features=unsafe_disable_continuation_linearity_check -- wast -W=exceptions,function-references,typed-continuations tests/misc_testsuite/typed-continuations/cont_twice.wast && test $? -eq 101) || test $? -eq 101
```

Comment:
Why `cargo run` and not `cargo test`? I thought the `cargo test` artifact would already have been built (maybe the `run` artifact hasn't/has too?).

Reply:
This particular test is now `#[ignore]`-d in the presence of `unsafe_disable_continuation_linearity_check`. Thus, you need to manually feed it into `wasmtime wast` to run it.

Reply:
Alternatively, the following works:
```
cargo test --features=unsafe_disable_continuation_linearity_check --test wast -- --include-ignored --exact Cranelift/tests/misc_testsuite/typed-continuations/cont_twice.wast
```

Reply:
In terms of avoiding additional building, it shouldn't make a difference anyway. As far as I can tell, this is the only place where we actually build with `unsafe_disable_continuation_linearity_check`, meaning that it will cause a separate rebuild of most stuff anyway.

Reply:
OK, I am fine with either. I was just curious. Thanks!